GoogleDataTransport/GDTCORLibrary/GDTCOREventMetricsCounter.m

```diff
@@ -154,13 +154,19 @@ + (BOOL)supportsSecureCoding {
   return YES;
 }
 
-- (void)encodeWithCoder:(nonnull NSCoder *)coder {
-  // TODO(ncooke3): Implement
+- (nullable instancetype)initWithCoder:(nonnull NSCoder *)coder {
+  self = [super init];
+  if (self) {
+    _droppedEventCounterByMappingID =
```

Comment:
Does "Metrics" mean anything besides "droppedEvents"?

Reply:
No. I realize this is super clear but I have a TODO checked in to rename it to `GDTCORDroppedEventCounter`.